packages/medusa/src/api/routes/admin/product-categories/update-product-category.ts

```diff
@@ -0,0 +1,126 @@
+import { IsOptional, IsString } from "class-validator"
+import { Request, Response } from "express"
+import { EntityManager } from "typeorm"
+
+import { ProductCategoryService } from "../../../../services"
+import { AdminProductCategoriesReqBase } from "../../../../types/product-category"
+import { FindParams } from "../../../../types/common"
+/**
+ * @oas [post] /product-categories/{id}
+ * operationId: "PostProductCategoriesCategory"
```

Comment:
question: should this be `Category` or `ProductCategory`
```suggestion
 * operationId: "PostProductCategoriesProductCategory"
```

Reply:
Based on the code conventions doc, it says this -

```
If a resource consists of multiple words you don’t have to repeat both in sub-segments. E.g.:

`GET /admin/sales-channels/:id` → `GetSalesChannelsChannel`
```